.babelrc

```diff
@@ -0,0 +1,7 @@
+{
+  "loose": ["all"],
+  "whitelist": [
+    "utility.inlineEnvironmentVariables"
```

Comment:
So for now, I am totally using babel just to inline environment variables. Will use other features down the road.